perftools/stresstesting/login-and-learn.yml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
@benjaoming the intention is to create a specific set of performance measurement tools for Kolibri. There's no benchmarking and profiling code right now, which means other devs have to rewrite the same tools and scripts over and over.

I might pull them out in a separate repo once they're more mature and generic, but i think this is a good place for now.